mrmustard/physics/gaussian_integrals.py

```diff
@@ -286,3 +287,83 @@ def contract_two_Abc(
     return complex_gaussian_integral(
         Abc, idx1, tuple(n + Abc1[0].shape[-1] for n in idx2), measure=-1.0
     )
+
+
+def complex_gaussian_integral_2(
```

Comment:
On first glance this method looks very similar to the single `Abc` case. I was wondering if this could be generalized to just handle both cases (maybe even more)? E.g. `complex_gaussian_integral(Abc: list[tuple], idx: tuple[int, ...], idx_conj: tuple[int, ...], measure: float = -1)` where `len(Abc)` determines how many Abc triples you're working with

Reply:
It is essentially the same for now, but Filippo has ideas to update it later (how the outer products are handled). One could maybe make one function to handle both cases.